tools/redpanda-gdb.py

```diff
@@ -601,16 +636,16 @@ def get_span(self, ptr):
 def find_storage_api(shard=None):
     if shard is None:
         shard = current_shard()
-    return gdb.parse_and_eval('::debug::app')['storage']['_instances'][
+    return gdb.parse_and_eval('debug::app')['storage']['_instances'][
```

Comment:
did this fix a bug? I think the argument for not doing this is that it wouldn't resolve if using the gdb python's `block` parameter to some methods which defines the lookup scope. but it doesn't really matter too much we can change it later if needed.

Reply:
With the leading ::, gdb was complaining that it was an invalid symbol name (same if using `info symbol` at cli)